sys/net/gnrc/pktbuf_static/include/pktbuf_static.h

```diff
@@ -40,6 +41,17 @@ typedef struct _unused {
     unsigned int size;      /**< the size of the unused section */
 } _unused_t;
 
+
+/**
+ * @brief   The actual static buffer used when module gnrc_pktbuf_static is used
+ *
+ * @warning This is an internal buffer and should not be touched by external
+ *          code
+ *
+ * @details This buffer is aligned to boundaries of `sizeof(_unused_t)`
+ */
+extern alignas(sizeof(_unused_t)) uint8_t gnrc_pktbuf_static_buf[CONFIG_GNRC_PKTBUF_SIZE];
```

Comment:
Why does the buffer need to be exposed anyways?

Reply:
That is a good question. Let's see how Murdock likes it if I make it `static`.

Reply:
because of this, apparently:

https://github.com/RIOT-OS/RIOT/blob/8cc31c21e196978bcb300f5852388312dc4cb1f6/sys/net/gnrc/pktbuf/include/pktbuf_internal.h#L64-L73

this is used in the common pktbuf code in an `assert()`, so it cannot easily be made `static`.

Reply:
Mhhh does this really need to be made `static inline`? Does anything speaks against putting it into the object code of `gnrc_pktbuf_static` and `gnrc_pktbuf_malloc` respectively? Seems to me also that that would make more sense anyways, as only the implementation really knows, if a pointer is in the packet buffer or not.

Reply:
I added a second commit that moves `gnrc_pktbuf_contains()` to `gnrc_pktbuf_static.c` and `gnrc_pktbuf_malloc.c` and makes the static buffer `static`. I expect that this will increase ROM a bit when LTO is not used and additionally `DEVELHELP` is set.